src/soundvectors.py

```diff
@@ -706,7 +718,10 @@ def validate(self, sound):
             if is_valid_sound(sound.name):
                 return sound.name
         elif sound and self.ts:
-            sound_transcribed = self.ts([sound])[0]
+            if isinstance(self.ts, TranscriptionSystem):
+                sound_transcribed = self.ts[sound]
```

Comment:
sound lookup the pyclts way.

Reply:
does this still work with `linse`?

Reply:
The next lines provide compat with `linse`. The `ifinstance` makes sure this line is only called with `pyclts.TranscriptionSystem` instances.

Reply:
Ah, perfect. Just saw that, the else statement was not included in the conversation overview :)